wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1338,213 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="RepeatInterval" type="xs:int" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>Optional time interval, in seconds, between audio clips when the audio clip is repeated.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="AudioOutputLevel" type="xs:int" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional level to control the volume of the audio clip when it is played.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="ScheduleToken" type="tt:ReferenceToken" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional schedule token associated with the audio clip configuration. The audio clip should be played when the configured schedule is triggered.</xs:documentation>
+					</xs:annotation>
+					</xs:element>					
+					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first ONVIF then Vendor -->
+				</xs:sequence>
+				<xs:anyAttribute processContents="lax"/>
+			</xs:complexType>
+			
+			<!--===============================-->
+			<xs:complexType name="GetAudioClipsResponseItem">
+				<xs:sequence>
+					<xs:element name="Token" type="tt:ReferenceToken">
+						<xs:annotation>
+							<xs:documentation>Unique token associated with the audio clip.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Configuration" type="tr2:AudioClip">
+						<xs:annotation>
+							<xs:documentation>Configuration of the audio clip.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="PlayStatus" type="xs:boolean" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Optional PlayStatus to indicate whether the audio clip is currently playing or not. PlayStatus should be false during the RepeatInterval when the audio clip is repeated.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+			<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
+				</xs:sequence>
+				<xs:anyAttribute processContents="lax"/>
+			</xs:complexType>
+			<!--===============================-->
+			
+			<xs:element name="GetAudioClips">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Token" type="tt:ReferenceToken" minOccurs="0">
+							<xs:annotation>
+								<xs:documentation>Audio clip token.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>			
+			<xs:element name="GetAudioClipsResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="AudioClipItem" type="tr2:GetAudioClipsResponseItem" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Wouldn't it better to use element name as "AudioClips" and its complex type name as "AudioClip", it will be with the same style as e.g. GetProfiles and all another Get commands.

Reply:
It is used similar to GetRecoridngJobs, GetRecoridngs.